packages/react-swipeable-views/src/SwipeableViews.js

```diff
@@ -775,7 +775,6 @@ class SwipeableViews extends Component {
       <div
         ref={(node) => { this.rootNode = node; }}
         style={Object.assign({}, axisProperties.root[axis], style)}
-        role="listbox"
```

Comment:
I think that we should keep that role. At least [bootstrap](https://v4-alpha.getbootstrap.com/components/carousel/) is going so.